packages/e2e/package.json

```
@@ -0,0 +1,91 @@
+{
+  "name": "@yoroi/e2e",
```

please create a folder `/tests/` -> `e2e-wallet-mobile`
the packages folder is for dependencies only
```suggestion
  "name": "@yoroi/e2e-wallet-mobile",
```

---

Done